server/server/settings.py

```diff
@@ -158,6 +158,13 @@
 SESSION_COOKIE_HTTPONLY = True
 CSRF_TRUSTED_ORIGINS = ['https://interview.swecc.org']
 
+devclient = 'http://localhost:5173'
+
+if DJANGO_DEBUG == True:
+    CORS_ALLOWED_ORIGINS.append(devclient)
+    CSRF_TRUSTED_ORIGINS.append(devclient)
+
+
 # PROD ONLY
 CSRF_COOKIE_SECURE = True
```

Comment:
Should make this `CSRF_COOKIE_SECURE = !DJANGO_DEBUG` as we don't want secure for local.

Reply:
Might want to change line 155 and 156 too to 

`CSRF_COOKIE_SAMESITE = 'None' if !DJANGO_DEBUG else 'Lax'`
`SESSION_COOKIE_SAMESITE = 'None' if !DJANGO_DEBUG else 'Lax'

Reply:
Or does it work locally without this? `COOKIE_SECURE` means it needs to be https

Reply:
It was working locally. Not sure why though

Reply:
Interesting, might be because it's same same site so it doesn't require https.